fhd_core/beam_modeling/beam_setup.pro

```diff
@@ -24,14 +24,14 @@ IF Keyword_set(transfer_psf) then begin
     if ~file_test(transfer_psf + '/' + obs.obsname + '_obs.sav') AND ~file_test(file_dirname(transfer_psf) $
       + '/metadata/' + obs.obsname + '_obs.sav') then $
       message, transfer_psf + '/' + obs.obsname + '_obs.sav or ' + file_dirname(transfer_psf) + $
-      '/metadata/' + obs.obsname + '_obs.sav not found during psf transfer. Beam_integral needed in PS.'
+      '/metadata/' + obs.obsname + '_obs.sav not found during psf transfer. primary beam_sq_area needed in PS.'
       
     psf = getvar_savefile(transfer_psf + '/' + obs.obsname + '_beams.sav','psf')
     if file_test(transfer_psf + '/' + obs.obsname + '_obs.sav') then $
       obs_restore = getvar_savefile(transfer_psf + '/' + obs.obsname + '_obs.sav','obs')
     if file_test(file_dirname(transfer_psf) + '/metadata/' + obs.obsname + '_obs.sav') then $
       obs_restore = getvar_savefile(file_dirname(transfer_psf) + '/metadata/' + obs.obsname + '_obs.sav','obs')
-    obs.beam_integral = pointer_copy(obs_restore.beam_integral)
+    obs.primary_beam_sq_area = pointer_copy(obs_restore.primary_beam_sq_area)
```

Comment:
Please check with Bryna that it is OK to change this name, for Eppsilon.

Reply:
I've submitted an issue on eppsilon. We talked about this change back in August.

Reply:
Yes, I recall talking about it, and I have no problem with the change. As long as it's compatible with Eppsilon, it's fine.